main/src/test/java/org/mobilitydata/gtfsvalidator/validator/TripTravelSpeedValidatorTest.java

```diff
@@ -0,0 +1,238 @@
+    TripTravelSpeedValidator underTest = new TripTravelSpeedValidator();
+    underTest.stopTable =
+        createStopTable(
+            noticeContainer,
+            ImmutableList.of(
+                createStop(2, "s0", 16.251233D, -61.591997D),
+                createStop(3, "s1", 16.251053D, -61.591241D),
+                createStop(4, "s2", 16.242191D, -61.588554D),
+                createStop(5, "s3", 16.249092D, -61.494831D)));
+
+    underTest.tripTable =
+        createTripTable(
+            noticeContainer, ImmutableList.of(createTrip(34, "route id", "service id", "t1")));
+
+    underTest.stopTimeTable =
+        createStopTimeTable(
+            noticeContainer,
+            ImmutableList.of(
+                createStopTime(
+                    4,
+                    "t1",
+                    "s0",
+                    0,
+                    GtfsTime.fromSecondsSinceMidnight(3130),
+                    GtfsTime.fromSecondsSinceMidnight(100)),
+                createStopTime(
+                    5,
+                    "t1",
+                    "s1",
+                    1,
+                    GtfsTime.fromSecondsSinceMidnight(5200),
+                    GtfsTime.fromSecondsSinceMidnight(5200)),
+                createStopTime(
+                    6,
+                    "t1",
+                    "s2",
+                    2,
+                    GtfsTime.fromSecondsSinceMidnight(5200),
+                    GtfsTime.fromSecondsSinceMidnight(5200)),
+                createStopTime(
+                    6,
+                    "t1",
+                    "s3",
+                    3,
+                    GtfsTime.fromSecondsSinceMidnight(5400),
+                    GtfsTime.fromSecondsSinceMidnight(5300))));
+
+    underTest.validate(noticeContainer);
+
+    assertThat(noticeContainer.getValidationNotices())
+        .containsExactly(new FastTravelBetweenStopsNotice("t1", 398.1887657929889D, List.of(1, 2, 3)));
+  }
+
+  @Test
+  public void noFastTravelShouldNotGenerateNotice() {
+    NoticeContainer noticeContainer = new NoticeContainer();
+    TripTravelSpeedValidator underTest = new TripTravelSpeedValidator();
+    underTest.stopTable =
+        createStopTable(
+            noticeContainer,
+            ImmutableList.of(
+                createStop(2, "s0", 16.251233D, -61.591997D),
+                createStop(3, "s1", 16.251053D, -61.591241D),
+                createStop(4, "s2", 16.242191D, -61.588554D)));
+
+    underTest.tripTable =
+        createTripTable(
+            noticeContainer, ImmutableList.of(createTrip(34, "route id", "service id", "t1")));
+
+    underTest.stopTimeTable =
+        createStopTimeTable(
+            noticeContainer,
+            ImmutableList.of(
+                createStopTime(
+                    4,
+                    "t1",
+                    "s0",
+                    0,
+                    GtfsTime.fromSecondsSinceMidnight(130),
```

Comment:
Could you please add tests with empty arrival and departure times as well (e.g., non-timepoints)?